controller/app.controller.js

```diff
@@ -39,38 +39,10 @@ exports.getArticleById = (req,res,next) => {
 
 
 
+exports.getArticles = (req,res,next) => {
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+  selectAllArticles().then((allArticlesArray) => {
+ 
+    res.status(200).send(allArticlesArray)
```

Comment:
We want to be sending all articles back here as an object on a descriptive key i.e. { articles: arrayOfArticles....} in keeping with JSON conventions